service/src/main/java/uk/gov/hmcts/reform/fpl/service/validators/RespondentsChecker.java

```diff
@@ -49,17 +49,18 @@ private static boolean isEmptyRespondent(Respondent respondent) {
         }
 
         return isEmptyAddress(respondentParty.getAddress())
-                && isEmptyTelephone(respondentParty.getTelephoneNumber())
-                && isEmptyEmail(respondentParty.getEmail())
-                && allEmpty(
-                respondentParty.getFirstName(),
-                respondentParty.getLastName(),
-                respondentParty.getDateOfBirth(),
-                respondentParty.getGender(),
-                respondentParty.getPlaceOfBirth(),
-                respondentParty.getRelationshipToChild(),
-                respondentParty.getContactDetailsHidden(),
-                respondentParty.getLitigationIssues());
+            && isEmptyTelephone(respondentParty.getTelephoneNumber())
+            && isEmptyEmail(respondentParty.getEmail())
+            && allEmpty(
+            respondentParty.getFirstName(),
+            respondentParty.getLastName(),
+            respondentParty.getDateOfBirth(),
+            respondentParty.getGender(),
+            respondentParty.getPlaceOfBirth(),
```

Comment:
place of birth can probably be removed from here now?

Reply:
yes 👍